src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.opationstrategy.OperationHandler;
+import core.basesyntax.opationstrategy.OperationStrategy;
+import core.basesyntax.opationstrategy.OperationStrategyImpl;
+import core.basesyntax.operations.BalanceOperation;
+import core.basesyntax.operations.FruitTransaction;
+import core.basesyntax.operations.PurchaseOperation;
+import core.basesyntax.operations.ReturnOperation;
+import core.basesyntax.operations.SupplyOperation;
+import core.basesyntax.shopserviceandreportgenerator.ReportGenerator;
+import core.basesyntax.shopserviceandreportgenerator.ReportGeneratorImpl;
+import core.basesyntax.shopserviceandreportgenerator.ShopService;
+import core.basesyntax.shopserviceandreportgenerator.ShopServiceImpl;
+import core.basesyntax.workwithfile.DataConverterImpl;
+import core.basesyntax.workwithfile.FileReaderImpl;
+import core.basesyntax.workwithfile.FileWriterImpl;
+import core.basesyntax.workwithfileinterface.DataConverter;
+import core.basesyntax.workwithfileinterface.FileDataReader;
+import core.basesyntax.workwithfileinterface.FileWriter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        FileDataReader fileReader = new FileReaderImpl();
+        List<String> inputReport = fileReader.read("IDontKnowFileName.csv");
```

Comment:
Let’s give a more informative name for this file. Also, place all files in the resources directory